components/common/src/command/package/install.rs

```diff
@@ -39,7 +39,6 @@ use std::fs;
 use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
-use ansi_term::Colour::{Blue, Green, Yellow};
```

Comment:
We'll still need it in the common crate's deps as that's our current coloring implementation in the [UI](https://github.com/habitat-sh/habitat/blob/db9caa8c5f704d38a227e2130dae1afca6f52ace/components/common/src/ui.rs#L18).

I think there's still a couple of spots in the hab crate that still use this crate to paint some errors, but I have those next in my sights. Hopefully when this is all over we only have 1 reference to `ansi_term` (in the common crate) which will be an internal implementation detail only (i.e. free to swap, change, update in the future). Closer!